From 5e755421d8558a13813aa8bcce385f1483ba5734 Mon Sep 17 00:00:00 2001
From: Thanh Quan <thanh.quan.xn@renesas.com>
Date: Tue, 27 May 2025 21:45:52 +0700
Subject: [PATCH 28/46] arm64: dts: renesas: r8a78000: Add device tree node for
 Multi-Protocol PHY

Add the device tree node for the Multi-Protocol PHY controller
on the R-Car X5H (R8A78000) SoC.

This enables support for initializing and controlling the MP-PHY
hardware used for high-speed serial interfaces such as PCIe and
Ethernet PCS.

Signed-off-by: Thanh Quan <thanh.quan.xn@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a78000.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a78000.dtsi b/arch/arm64/boot/dts/renesas/r8a78000.dtsi
index 375e6a7e4d1f..6a4d54dcf8db 100644
--- a/arch/arm64/boot/dts/renesas/r8a78000.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a78000.dtsi
@@ -926,6 +926,13 @@ rst: reset-controller@c1320000 {
 			compatible = "renesas,r8a78000-rst";
 			reg = <0 0xc1320000 0 0x4000>;
 		};
+
+		mp_phy: mp_phy@c9a00000 {
+			compatible = "renesas,multi-protocol-phy";
+			reg = <0 0xc9a00000 0 0x100000>;
+			#phy-cells = <2>;
+			status = "disabled";
+		};
 	};
 
 	timer {
-- 
2.34.1

